ts/index.ts

```diff
@@ -319,7 +318,7 @@ class Gossipsub extends Pubsub {
         }
       }
     }
-    this.outbound.set(p, outbound)
+    this.outbound.set(p.id.toB58String(), outbound)
```

Comment:
@tuyennhv @wemeetagain Is this ok? types aren't matching since map is string => boolean, while p is PeerStreams

Reply:
this looks good to me, I see `this.outbound` works with `peerId.toB58String` as key in other places